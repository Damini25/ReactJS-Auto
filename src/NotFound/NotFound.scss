.notFound{
    height: 90%;
    padding: 20% 20%;
    text-align: center;
    .logo1{
        img{
            width: 40%;
        }
    }
    a{
        text-decoration: none;
    }
}