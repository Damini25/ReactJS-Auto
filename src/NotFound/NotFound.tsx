import React from 'react';
import logo from '../autologo.png'
import './NotFound.scss';
import { Link } from "react-router-dom";

function NavBar() {
  return (
    <div className="notFound">
      <div className="logo1"><img src={logo} className="appLogo" alt="logo" /></div>
      <div className="desc">
        <p className="font32 fontbold">404-Not Found</p>
        <p className="font18">Sorry the page you are looking for doesnot exist.</p>
        <p className="font18">You can always go back to the <Link to={`/`} className="ctacolor">homepage</Link>.</p>
      </div>
    </div>

  );
}

export default NavBar;
