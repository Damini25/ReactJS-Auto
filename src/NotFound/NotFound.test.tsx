import React from 'react';
import { render, cleanup } from '@testing-library/react';
import { BrowserRouter as Router} from 'react-router-dom';
import NotFound from './NotFound';

afterEach(cleanup)
test('render <NotFound>', () => {
    const notfound = render(<Router><NotFound /></Router>)
    expect(notfound).toMatchSnapshot()
})