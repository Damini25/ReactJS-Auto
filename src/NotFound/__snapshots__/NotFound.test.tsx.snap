// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render <NotFound> 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="notFound"
      >
        <div
          class="logo1"
        >
          <img
            alt="logo"
            class="appLogo"
            src="autologo.png"
          />
        </div>
        <div
          class="desc"
        >
          <p
            class="font32 fontbold"
          >
            404-Not Found
          </p>
          <p
            class="font18"
          >
            Sorry the page you are looking for doesnot exist.
          </p>
          <p
            class="font18"
          >
            You can always go back to the 
            <a
              class="ctacolor"
              href="/"
            >
              homepage
            </a>
            .
          </p>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="notFound"
    >
      <div
        class="logo1"
      >
        <img
          alt="logo"
          class="appLogo"
          src="autologo.png"
        />
      </div>
      <div
        class="desc"
      >
        <p
          class="font32 fontbold"
        >
          404-Not Found
        </p>
        <p
          class="font18"
        >
          Sorry the page you are looking for doesnot exist.
        </p>
        <p
          class="font18"
        >
          You can always go back to the 
          <a
            class="ctacolor"
            href="/"
          >
            homepage
          </a>
          .
        </p>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
